### PR TITLE
[patch] Added additional ca.crt field to rsl secret

### PR DESCRIPTION
**Pull request by byalif**
## Issue

/MASAIB-1135

## Description

Create RSL_CA changes to add ca.crt for kmodels purposes. (AI Service operator has also been updated with the correct CRD to make this change work.)

## Test Results
I have tested with many edge cases on techzone cluster.

<hr/>

### :warning: Notes for Reviewers
* Ensure you have understood the PR guidelines in the Playbook before proceeding with a review.
* Ensure all sections in the PR template are appropriately completed.

### CODE REVIEW

File: ibm/mas_devops/roles/aibroker_tenant/tasks/config_rsl/main.yml [hunk 1]
```diff
@@ -1,7 +1,22 @@
 ---
+- name: Check if RSL CA certificate file exists
+  stat:
+    path: "{{ mas_config_dir }}/rsl/ca.crt"
```
Comment (karol-czarnecki) [inline ibm/mas_devops/roles/aibroker_tenant/tasks/config_rsl/main.yml:4]: why do we check if cert exists? What is other scenario?

File: ibm/mas_devops/roles/aibroker_tenant/tasks/config_rsl/main.yml [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+  register: rsl_ca_file_result
+
+- name: Read RSL CA cert from file
+  ansible.builtin.set_fact:
+    rsl_ca_crt: "{{ lookup('file', '{{ mas_config_dir }}/rsl/ca.crt') }}"
```
[the remark this answers is not in the document]
Comment (jasmin-30) [reply ibm/mas_devops/roles/aibroker_tenant/tasks/config_rsl/main.yml]: Updated code to only read rsl_ca_crt from RSL_CA_CRT env var.

File: ibm/mas_devops/roles/aibroker_tenant/templates/aibroker/aibrokerworkspace.yml.j2 [hunk 3]
```diff
@@ -42,6 +42,11 @@ spec:
             url: {{ rsl_url }}
             orgId: {{ rsl_org_id }}
             secretName: {{ mas_aibroker_rsl_secret }}
+{% if mas_app_channel == '9.1.x' %}
```
[the remark this answers is not in the document]
Comment (jasmin-30) [reply ibm/mas_devops/roles/aibroker_tenant/templates/aibroker/aibrokerworkspace.yml.j2]: We don't need this condition anymore as this was not part of 9.1.x channel. We will always encode certificate in CR.

**Review by karol-czarnecki — APPROVED** [review]

**Comment by karol-czarnecki** [comment]
lgtm
